src/averageAges.js

```diff
@@ -1,64 +1,74 @@
 'use strict';
 
 /**
- * Implement calculateMenAverageAge function
- *
- * Function returns average age of men in array. If `century` is specified then
- * function calculates average age only for men who died in this century
- *
- * To calculate century:
- * Divide year of person's death by 100: Math.ceil(person.died / 100)
- *
  * @param {object[]} people
  * @param {number} century - optional
  *
  * @return {number}
  */
+
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const isMale = person => person.sex === 'm';
+
+  const man = people.filter((person) =>
```

Comment:
Move all helpers outside as it's not a good practice to create functions inside a function.
isMale looks like a result instead of a function. I think a better name is checkIsMale or something like this.
Also, man is only one, men - plural